eng/StyleCop.Analyzers.ruleset

```diff
@@ -69,6 +69,7 @@
     <Rule Id="SA1136" Action="Error" />
     <Rule Id="SA1137" Action="Error" />
     <Rule Id="SA1139" Action="Error" /> 
+	<Rule Id="SA1167" Action="Error" /> <!-- Elements must have same indentation -->
```

Comment:
this element (ironically) has different indentation 😄

Reply:
We need stylecop rules for the rule sets...